docs/pages/upgrading/cloud-linux.mdx

```diff
@@ -0,0 +1,179 @@
+  current version of the Auth Service and Proxy Service by running the following
+  command, where `mytenant` is the name of your Teleport Team or Teleport
+  Enterprise Cloud account. This requires the [`jq` CLI
+  tool](https://jqlang.github.io/jq/):
+
+  ```code
+  $ curl -s https://mytenant.teleport.sh/webapi/ping | jq '.server_version'
+  ```
+
+Teleport supports automatic agent updates for systemd-based Linux distributions
+using `apt`, `yum`, and `zypper` package managers. The [Automatic Update
+Architecture](../architecture/agent-update-management.mdx) guide describes
+how agent updating works. Automatic agent upgrades require:
+
+- A Teleport Cloud account that supports automatic upgrades. To determine if
+  your account supports automatic upgrades, run the following command, replacing
+  `mytenant.teleport.sh` with the address of your Teleport Cloud account:
+
+  ```code
+  $ curl -s https://mytenant.teleport.sh/webapi/ping | jq '.automatic_upgrades'
+  true
+  ```
+
+- At least one Teleport Enterprise agent, started via systemd on a distribution
+  using the `apt` or `yum` package managers. 
+
+  So you can access each agent, we assume that each agent runs the Teleport SSH
+  Service in addition to any other services. See the [Deploy Teleport Agents
+  with Terraform](../agents/deploy-agents-terraform.mdx] guide for an example.
+
+## Step 1/2. Find agents to upgrade
+
+To list agents that are not enrolled in automatic upgrades, use the `tctl
+inventory ls` command with the `--upgrader=none` flag. This command also uses
+the `--services` flag to constrain the search to Teleport processes running
+agent services:
+
+```code
+$ tctl inventory ls \
+  --services=node,kube,db,app,windows_desktop,discovery,okta \
+  --upgrader=none
+Server ID                            Hostname      Services Version Upgrader
+------------------------------------ ------------- -------- ------- --------
+00000000-0000-0000-0000-000000000000 ip-10-1-6-130 Node     v13.4.5 none
+```
+
+If you have a lot of agents on different versions and want to prioritize enrolling
+your oldest agents, you can limit your search using the `--older-than` filter:
+
+```code
+$ tctl inventory ls --upgrader=none --older-than=v1.2.3
+Server ID                            Hostname        Services Version Upgrader
+------------------------------------ --------------- -------- ------- --------
+00000000-0000-0000-0000-000000000000 old.example.com Node     v1.1.1  none
+...
+```
+
+When you enroll each agent in automatic upgrades in the next section, you can
+run the following commands to fetch the hostname for each agent and access it
+via Teleport. 
+
+For each agent ID returned by the `tctl inventory ls` command, copy the ID and
+run the following `tctl` command to get the agent's host name and access the
+host via `tsh`:
+
+```code
+$ HOST=00000000-0000-0000-0000-000000000000
+$ USER=root
+$ tsh ssh ${USER?}@$(tctl get node/${HOST?} --format json | jq -r '.[0].spec.hostname')
```

Comment:
You can just `tsh ssh <user>@<uuid>` without having to figure out a hostname for the node.
```suggestion
$ tsh ssh "${USER?}@${HOST?}"
```